kratos/processes/integration_values_extrapolation_to_nodes_process.cpp

```diff
@@ -378,39 +368,36 @@ void IntegrationValuesExtrapolationToNodesProcess::InitializeVariables()
 
     // The list of nodes
     auto& r_nodes_array = mrModelPart.Nodes();
-    const auto it_node_begin = r_nodes_array.begin();
 
     // Initialize values
-    #pragma omp parallel for
-    for(int i = 0; i < static_cast<int>(r_nodes_array.size()); ++i) {
-        auto it_node = it_node_begin + i;
+    block_for_each(r_nodes_array, [&](Node<3>& rNode){
 
         // We initialize the doubles values
         for ( const auto p_var : mDoubleVariable) {
-            if (mExtrapolateNonHistorical) it_node->SetValue(*p_var, 0.0);
-            else it_node->FastGetSolutionStepValue(*p_var) = 0.0;
+            if (mExtrapolateNonHistorical) rNode.SetValue(*p_var, 0.0);
```

Comment:
@philbucher I tried using Variable Utils for all 4 loops here, but was showing some error. _"No matching Member found"_

Reply:
what exactly was the error? (just curious)

I think we should anyway refactor to not have the `if` in every iteration
i.e. sth like 
~~~cpp
if (mExtrapolateNonHistorical) {
    block_for_each(
    // use SetValue ...
    );
} else {    
    // use FastGetSolutionStepValue...
    block_for_each(
    );
}

~~~